streams/readable-streams/pipe-through.js

```diff
@@ -98,11 +98,9 @@ test(() => {
   };
 
```

Comment:
I think this file should get merged with piping/pipe-through one day. Not today.